.gitlab-ci.yml

```diff
@@ -0,0 +1,32 @@
+stages:
+  - build
+  - deploy
+
+build:
+  tags: ["runner:docker", "size:large"]
```

Comment:
How do I found out about sizes other than `size:large`?

Reply:
I'll respond in internal slack - not to share internal resource links